rel/files/mongooseim.toml

```diff
@@ -211,64 +211,64 @@
   periodic_report = 10_800_000
 
 [modules.mod_adhoc]
-{{#mod_amp}}
 
+{{#mod_amp}}
 {{{mod_amp}}}
-{{/mod_amp}}
 
+{{/mod_amp}}
 [modules.mod_disco]
   users_can_see_hidden_services = false
 
 [modules.mod_commands]
-{{#mod_cache_users}}
 
-[modules.mod_cache_users]{{{mod_cache_users}}}
+{{#mod_cache_users}}
+[modules.mod_cache_users]
+{{{mod_cache_users}}}
```

Comment:
This disables this module in the prod config.

Reply:
It is only a whitespace change, it cannot disable anything.

Reply:
It was disabled before and remains unchanged. Let's change this in a separate PR.